### DISCUSSION TITLE
Quick Question + THANK YOU!

### DISCUSSION THREAD

**Discussion by armenr** [chadgeary/cloudblock#16] [Q&A] [answered]
Will this setup also filter HTTP/HTTPS traffic and block embeds/JS/HTML as well, or only DNS-level blocking?

**Comment by chadgeary** [comment, marked as answer]
Hey @armenr - Pihole is DNS-based ad blocking. Something client side would be needed for the others.

Though it’s not filter related - there is a variable (vpn_traffic) that can be set to just dns (default) or all traffic.
